dashboard/src/routes/BuildDetails/BuildDetails.tsx

```diff
@@ -151,7 +153,12 @@ const BuildDetails = (): JSX.Element => {
   //TODO: loading and 404
   if (!data || error) return <span></span>;
 
-  return <SectionGroup sections={sectionsData} />;
+  return (
+    <>
+      <SectionGroup sections={sectionsData} />
+      <BuildDetailsTestSection />
```

Comment:
can you add an error boundary here? there is too many things that can go wrong in a datatable
https://www.npmjs.com/package/react-error-boundary

or you can define one by yourself 

https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/error_boundaries/